tests/rust/.gitignore

```diff
@@ -0,0 +1,20 @@
+# Created by https://www.toptal.com/developers/gitignore/api/rust
```

Comment:
maybe I am missing smth but in this folder, there is only .gitignore and cargo.toml, did you forgot to add smth?

Reply:
I started with that branch yesterday and thought I could merge master into it...which was a bad idea. This folder should not exist...that's why I closed the PR. I'll make a new one